src/client.ts

```diff
@@ -7,16 +7,28 @@ import { version } from "../package.json";
 import { ManagementController } from "./facades/management";
 
 export class EdcConnectorClient {
-  readonly management: ManagementController;
-  readonly observability: ObservabilityController;
-  readonly public: PublicController;
+
+  apiToken: string | undefined;
+  addresses: Addresses = {};
+  inner = new Inner();
```

Comment:
they were already like this, I'm not sure if it was by purpose or not, anyway I'm always in for some encapsulation

Reply:
woa no, they cannot be private otherwise the builder cannot work correctly, in java the builder pattern works with inner class, IDK if this is possible in TS as well